File: docs/fundamentals/runtime-libraries/snippets/System/Double/Overview/csharp/representation2.cs
// <Snippet4>
using System;

public class Example14
{
    public static void Main()
    {
        Double value = 123456789012.34567;
        Double additional = Double.Epsilon * 1e15;
        Console.WriteLine($"{value} + {additional} = {value + additional}");
    }
}
// The example displays the following output:
//    123456789012.346 + 4.94065645841247E-309 = 123456789012.346
// </Snippet4>

File: docs/fundamentals/runtime-libraries/snippets/System/Double/Overview/vb/representation2.vb
' Visual Basic .NET Document
Option Strict On

' <Snippet4>
Module Example15
    Public Sub Main()
        Dim value As Double = 123456789012.34567
        Dim additional As Double = Double.Epsilon * 1.0E+15
        Console.WriteLine("{0} + {1} = {2}", value, additional,
                                           value + additional)
    End Sub
End Module
' The example displays the following output:
'   123456789012.346 + 4.94065645841247E-309 = 123456789012.346
' </Snippet4>

